tests/testGetId.py

```diff
@@ -22,23 +22,23 @@
 # see <https://www.lsstcorp.org/LegalNotices/>.
 #
 
+import os
 import unittest
 import lsst.utils.tests
 
 import lsst.daf.persistence as dafPersist
-from lsst.obs.sdss import SdssMapper
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
 
 
 class GetIdTestCase(lsst.utils.tests.TestCase):
     """Testing butler exposure id retrieval"""
 
     def setUp(self):
-        self.bf = dafPersist.ButlerFactory(mapper=SdssMapper(root="."))
-        self.butler = self.bf.create()
+        self.butler = dafPersist.Butler(inputs=os.path.split(ROOT)[0])
```

Comment:
Should this be a temp directory rather than an existing directory, given that a yaml file is created that seems to in theory have an effect the next time the test is run?